projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1884,8 +1898,11 @@ private boolean listTestrigs(
     return true;
   }
 
-  private String loadQuestion(Path file, Map<String, String> bfq) {
-    String questionText = CommonUtil.readFile(file);
+  private String loadQuestion(
+      Path file, String questionText, String questionKey, Map<String, String> bfq) {
+    if (questionText == null) {
```

Comment:
Arguments that can be `null` should have `Nullable` annotations.

Reply:
This function seems to have gotten a lot more complex. Add javadoc to explain what it does especially with multiple optional arguments.